multi_gpu/env.yml

```
@@ -0,0 +1,10 @@
+channels:
+- pytorch-nightly
```

```suggestion
- pytorch
```
Can we avoid pytorch-nightly? That doesn't seem stable for example code.